source/Calamari.Aws/Deployment/Conventions/UploadAwsS3Convention.cs

```diff
@@ -122,6 +115,18 @@ public void Install(RunningDeployment deployment)
             }
         }
 
+        private void SetOutputVariables(RunningDeployment deployment, IEnumerable<S3UploadResult> results) 
+        {
+            Log.SetOutputVariable(SpecialVariables.Package.Output.FileName, Path.GetFileName(deployment.PackageFilePath));
+            Log.SetOutputVariable(SpecialVariables.Package.Output.FilePath, deployment.PackageFilePath);
+            foreach (var result in results)
+            {
+                if (!result.IsSuccess()) continue;
+                Log.Info($"Saving object version id to variable \"Octopus.Action[{deployment.Variables["Octopus.Action.Name"]}].Output.Files[{result.BucketKey}]\"");
```

Comment:
Output variable saving text has been changed to mention that we are saving the version id as per [discussion in slack](https://octopusdeploy.slack.com/archives/C27FNL3QW/p1524169201000339)